Simplified/AxisService/NYPLAxisService.swift

```diff
@@ -81,12 +89,203 @@ class NYPLAxisService: NSObject {
   }
   
   /// Fulfill AxisNow license. Notifies NYPLBookDownloadBroadcasting upon completion or failure.
+  
   /// - Parameters:
   ///   - downloadTask: downloadTask download task
   @objc func fulfillAxisLicense(downloadTask: URLSessionDownloadTask) {
+    DispatchQueue.global(qos: .utility).async {
+      let dispatchGroup = DispatchGroup()
```

Comment:
Can the DispatchGroup be a property of the `NYPLAxisService ` class? So we don't need to pass it  to every function.

Reply:
Agreed. Done